app/build.gradle

```diff
@@ -30,6 +30,8 @@ android {
         } catch (Exception ignore) {
         }
         buildConfigField "String", "COMMIT_HASH", ('"' + commit + '"')
+        buildConfigField "String", "API_KEY", '"JRJPPWC6ZA7DKKTSU2R3"'
+        buildConfigField "String", "API_SECRET", '"7$$67JtrfkSYtAncGBEaJp$v$Y9$ZJUzYVy8GuBm"'
```

Comment:
This does not make clear which API key is meant. Could you please use something like `PODCASTINDEX_API_KEY`?

Reply:
Done